docs/chapters/baseplatform/deploying-the-image.rst

```diff
@@ -6,4 +6,6 @@
 Image names for PELUX targets
 -----------------------------
 * Intel NUC and ARP with Intel SMARC: ``core-image-pelux-intel-corei7-64.wic``
-* Raspberry Pi 3: ``core-image-pelux-raspberrypi3.rpi-sdimg``
+* Raspberry Pi 3: ``core-image-pelux-raspberrypi3.wic``
+
+.. note:: Image names depend on the built image and the image might contain suffixes such as ``-dev``. An image is always either a ``-minimal`` or ``-qtauto-neptune`` image. For software update images, a ``.swu`` file exists.
```

Comment:
why do we need this here?

Reply:
Just information to know that wic is the actual image for both of them. Its not mentioned anywhere else.

Reply:
that makes no sense: it's just a variant. why would we have different image formats for different variants?

Reply:
We dont have different formats for variants.. all our images are wic only. Wic is not mentioned anywhere tho.

Reply:
read my comment again

Reply:
I read it! 
>why would we have different image formats for different variants?
We dont! We are simply pointing that wic is our format for all the images? I dont see the issue here.. Can you elaborate your thoughts?

Reply:
Yes. Remove the comment. It's imho useless.

Reply:
Oleksandr, i dont understand what you mean. I think u misunderstood something. Can you please re-review it from the beginning. 

What im trying to say, is that we have different images, based on variants.. we can have an image called core-image-pelux-minimal-dev-intel-corei7-64.wic and an image called core-image-pelux-minimal-intel-corei7-64.wic.. they are still the same format, its just different name.

Reply:
The section is about deploying images, not about image variants.